src/components/Badge/Badge.module.css

```diff
@@ -0,0 +1,21 @@
+.shared {
+  @apply rounded-med font-inter  text-white text-12 py-xxxs px-xs;
+}
```

Comment:
you almost close from the design but as per the depicted image we need to adjust the values of `font-weight` and `line-height`
![Capture](https://user-images.githubusercontent.com/8650435/180893718-1c11932a-71a6-473e-8cfe-1cd83e4466b2.PNG)